apps/dash-image-segmentation/app.py

```diff
@@ -322,9 +370,10 @@ def annotation_react(
     sigma_range_slider_value,
     masks_data,
     segmentation_data,
-    classifier_store_data,
```

Comment:
these two objects are either recomputed from scratch or not changed so they are not needed as State. Removing them reduces the amount of data to be shipped over the network, resulting in a significant performance improvement as can be seen on the playground deployment.